rest/src/main/java/de/starwit/rest/controller/AppSetupController.java

```diff
@@ -28,14 +30,14 @@
   private AppSetupService appSetupService;
 
   @Operation(summary = "Creates new App from templates.")
-  @PostMapping(value = "/setupApp")
-  public void generateApp(@RequestBody GeneratorDto dto) throws NotificationException {
+  @PostMapping(value = "/setupapp")

```

Comment:
Why change the path to all lowercase?

Reply:
because of convetion - see for example here: https://restfulapi.net/resource-naming/